go/vt/wrangler/traffic_switcher.go

```diff
@@ -704,6 +707,48 @@ func (wr *Wrangler) dropArtifacts(ctx context.Context, sw iswitcher) error {
 	return nil
 }
 
+// finalizeMigrateWorkflow deletes the streams for the Migrate workflow.
+// We only cleanup the target for external sources
+// FIXME: implement dryRun
```

Comment:
is the `FIXME` still relevant? I do see `if dryRun` logic below.

Reply:
you are right, removed